README.md

```diff
@@ -19,20 +19,23 @@ Run psql command:
 
 ```
     postgres=# select * from pg_log_errors_stats();
-     time_interval |  type   |       message        | count
```

Comment:
May be add "## Testing" block
Just an example of how to run tests

Reply:
i've add basic description, there is not a much to write about